shortcut.go

```diff
@@ -0,0 +1,69 @@
+package fyne
+
+import (
+	"sync"
+)
+
+// ShortcutHandler is a default implementation of the shortcut handler
+// for the canvasObject
+type ShortcutHandler struct {
+	mu    sync.RWMutex
+	entry map[string]func(Shortcuter)
+}
+
+// HandleShortcut handle the registered shortcut
+func (sh *ShortcutHandler) HandleShortcut(shortcut Shortcuter) {
+	if shortcut == nil {
+		return
+	}
+	if sc, ok := sh.entry[shortcut.Shortcut()]; ok {
+		sc(shortcut)
+	}
+}
+
+// AddShortcut register an handler to be executed when ShortcutName command is triggered
+func (sh *ShortcutHandler) AddShortcut(shortcut Shortcuter, handler func(shortcut Shortcuter)) {
+	sh.mu.Lock()
+	defer sh.mu.Unlock()
+	if sh.entry == nil {
+		sh.entry = make(map[string]func(Shortcuter))
+	}
+	sh.entry[shortcut.Shortcut()] = handler
+}
+
+// Shortcuter is the interface implemented by values with a custom formatter.
+// The implementation of Format may call Sprint(f) or Fprint(f) etc.
+// to generate its output.
+type Shortcuter interface {
```

Comment:
I think that, now we have fewer related names this could now be called "Shortcut" (I thin it reads better).
Then perhaps the Shortcut() method is perhaps ShortcutName()? (or Id)